runtime/kernel/pcie/pcie_irq.c

```diff
@@ -42,7 +42,7 @@ int pcie_irqs_init(struct tlkm_device *dev)
 	struct tlkm_pcie_device *pdev =
 		(struct tlkm_pcie_device *)dev->private_data;
 
-	int ret = 0, irqn, err[NUMBER_OF_INTERRUPTS];
+	int ret = 0, irqn, err[NUMBER_OF_INTERRUPTS] = { [0 ... NUMBER_OF_INTERRUPTS - 1] = 1 };
```

Comment:
In a separate commit please.

Reply:
Rebasing seems impossible at this point so I don't think I can do this.